File: Udemy/Algoritmos e Logica de Programacao/Python/Matrizes/ProblemaNegativosMatriz.py
'''Problema "negativos_matriz"
Ler dois números M e N (máximo = 10), e depois ler uma matriz MxN de números inteiros, conforme
exemplo. Em seguida, mostrar na tela somente os números negativos da matriz.'''

m = int(input("Qual a quantidade de linhas da matriz? "))
n = int(input("Qual a quantidade de colunas da matriz? "))
vet : [[int]] = [[0 for x in range(n)] for x in range(m)]

for i in range(m):
    for j in range(n):
        vet[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("VALORES NEGATIVOS: ")
for i in range(m):
    for j in range(n):
        if vet[i][j] < 0:
            print(vet[i][j])

File: Algoritmos e Logica de Programacao/C/Matrizes/ProblemaNegativosMatriz.c
/*
Problema "negativos_matriz"
Ler dois números M e N (máximo = 10), e depois ler uma matriz MxN de números inteiros, conforme
exemplo. Em seguida, mostrar na tela somente os números negativos da matriz.
*/

#include<stdio.h>
#include<stdlib.h>
int main (){
	
	printf("Qual a quantidade de linhas da matriz? ");
	int m = 0;
	scanf("%d",&m);
	printf("Qual a quantidade de colunas da matriz? ");
	int n=0;
	scanf("%d",&n);
	int i =0;
	int j=0;
	int vet[m][n];
	for(i=0;i<m;i++){
		for(j=0;j<n;j++){
		printf("Elemento [%d,%d]: ",i,j);
		scanf("%d",&vet[i][j]);
		}
	}
	
	printf("VALORES NEGATIVOS: \n");
	
	for(i=0;i<m;i++){
		for(j=0;j<n;j++){
			if(vet[i][j]<0){
				printf("%d\n",vet[i][j]);
			}
		}
	}
	
	

    return 0;
}
